integ/lib/testing-tier.ts

```diff
@@ -79,6 +81,7 @@ export abstract class TestingTier extends Stack {
       vpc: this.vpc,
       subnetSelection: { subnetGroupName: NetworkTier.subnetConfig.testRunner.name },
       instanceType: new InstanceType('t3.small'),
+      machineImage: this.getMachineImageForDeadlineVersion(this.deadlineVersion),
```

Comment:
Just a question, but is this a change we just want to do in tests, or should we be putting this in the actual RFDK constructs as well?

Reply:
Right, we should check to see if there are any constructs that I missed.

I was surprised that I didn't find any, but it makes some sense: we're using an AMI for the Worker, and containers for the RCS and License Forwarder, so none of those are running directly on the `machineImage`.